auacm/angular/README.md

```diff
@@ -0,0 +1,8 @@
+# Angular 2 Rewrite
+
+This is just a simple overview of what is happening in this file. 
+
+First you are going to need to get Angular-Cli. This nifty tool will generate
```

Comment:
I will move the instructions to the main project's README and add the dependencies to the configuration scripts.